RestApiHomeTask/Controllers/ItemController.cs

```diff
@@ -0,0 +1,120 @@
+using Microsoft.AspNetCore.Mvc;
+using Npgsql;
+using RestApiHomeTask;
+
+namespace RestApiHomeTask.Controllers;
+
+[ApiController]
+[Route("api/[controller]/[action]")]
+[Consumes("application/json")]
+public class ItemController : ControllerBase
```

Comment:
I suggest creating service class for each controller that will contain all business logic, instead of having it directly in controller methods, as well as moving sql related code to corresponding repositories, so code become testable and maintainable

Reply:
I have update last commit this changes

Reply:
Hi Valery, I have try to fix last commit this changes.